.swiftlint.yml

```diff
@@ -43,6 +43,7 @@ disabled_rules:
   - nesting
   - no_fallthrough_only
   - notification_center_detachment
+  - optional_data_string_conversion
```

Comment:
This rule has been disabled for now because it produced over 50 errors. I'm going to fix it on a separate PR.